src/02-component-proof-of-concept.java

```diff
@@ -0,0 +1,127 @@
+import components.simplewriter.SimpleWriter;
+import components.simplewriter.SimpleWriter1L;
+import components.map.Map;
+
+public class Tracker {
+
+    private Map<String, Double> trackMap;
+
+    // Constructor
+
+    public Tracker() {
+
+        this.trackMap = new Map<>();
+
+    }
+
+    // Kernel Method 1: Add a new track (type, amount)
+
+    public void addTrack(String type, double amount) {
+
+        this.trackMap.add(type, amount);
+
+    }
+
+    // Kernel Method 2: Remove a track by type
+
+    public void removeTrack(String type) {
+
+        if (this.trackMap.hasKey(type)) {
+
+            this.trackMap.remove(type);
+
+        } else {
+
+            out.println("Type not found in the tracker.");
```

Comment:
I don't usually recommend having prints in methods like this. Either throw an error to the user or let the method fail silently. If you want to print something like this, look into logging. That way, the user can turn the logs on if they want them.